src/main/java/com/example/rentabookrestservices/dto/BookCreateDto.java

```diff
@@ -0,0 +1,67 @@
+package com.example.rentabookrestservices.dto;
+
```

Comment:
JavaDoc ekleeye gayret edelim

Reply:
tamamdır abi, şimdiden eklemelere başlayacağım inşallah